tsconfig.json

```diff
@@ -0,0 +1,62 @@
+{
+  "include": [
+    "./src/*.ts",
+    "./src/**/*.ts",
+    "./custom_typings/*.d.ts"
+  ],
+  "compilerOptions": {
+    /* Basic Options */
+    "target": "es5", /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
```

Comment:
You should be able to do ES2015 at the least.

Reply:
Done.